### ISSUE TITLE
Restart takes a long time when no notification daemon running

### ISSUE THREAD

**Issue by jfchevrette** [qtile/qtile#1749]
# Issue description

I was troubleshooting why `lazy.restart` would result in an unresponsive qtile for about 15 seconds. After a while I noticed the same behavior when running validate_config

`$ qtile-cmd -o cmd -f validate_config`

The following is logged:
```
2020-05-25 07:44:57,664 ERROR libqtile utils.py:send_notification():L234 g-io-error-quark: Error calling StartServiceByName for org.freedesktop.Notifications: Timeout was reached (24)
```

After starting a notification daemon (`dunst` in this case), things seem back to normal. 

# Qtile version

```
$ qtile --version
0.15.1.dev121+gdb0f044f
```
# Configuration

Default configuration

**Comment by jfchevrette** [comment]
Could qtile check to ensure there is a notification daemon running before it attempt to send one?

Without a notification daemon
```
>>> Notify.get_server_info()
(False, ret_name=None, ret_vendor=None, ret_version=None, ret_spec_version=None)
```

With dunst running
```
>>> Notify.get_server_info()
(True, ret_name='dunst', ret_vendor='knopwob', ret_version='1.4.1 (2019-07-03)', ret_spec_version='1.2')
```

**Comment by ramnes** [comment]
Eh, this is interesting. I think we should just fire and forget notifications somehow.

**Comment by ramnes** [comment]
That being said, shouldn't we have our own server running anyway?

**Comment by m-col** [comment]
Some people might not have a notification server running so it makes sense to check if one is registered with dbus with `Notify.get_server_info()` before sending the notification request. I'm surprised it wouldn't 'fire and forget' even in the absence of a dbus service.

> That being said, shouldn't we have our own server running anyway?

If you mean the one in `libqtile.notify` then I think that would only become active if it is imported, which would be the desirable approach for those who use standalone servers like dunst.

**Comment by m-col** [comment]
This should be fixed by #1775.
